### ISSUE TITLE
New About Panel: Broken Links

### ISSUE THREAD

**Issue by simonai1254** [mRemoteNG/mRemoteNG#2117]
## Expected Behavior
When opening the about panel and clicking on a link (e.g. Changelog), the web site containing the license document should open.

## Current Behavior
When opening the about panel and clicking on a link (e.g. License), an unhandled exception occurres. The exception shows the following message:

```
Exception Message
"An error occurred trying to start process 'https://github.com/mRemoteNG/mRemoteNG/blob/develop/CHANGELOG.md' with working directory 'C:\Program Files\mRemoteNG'. The system cannot find the file specified."

Stack trace
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(String fileName)
   at mRemoteNG.UI.Forms.FrmAbout.llChangelog_LinkClicked(Object sender, LinkLabelLinkClickedEventArgs e)
   at System.Windows.Forms.LinkLabel.OnLinkClicked(LinkLabelLinkClickedEventArgs e)
   at System.Windows.Forms.LinkLabel.OnMouseUp(MouseEventArgs e)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Label.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)

```

## Possible Solution
Fix the way how the link is tried to be opened.
Also please note that the "credits" links is no longer existing and the copyright should be updated.

Most likely this bug was introduced with Pull Request 1767

## Steps to Reproduce (for bugs)
1. Open Help > About
2. Click one of the links in the Pop-Up (e.g. "Changelog")
3. Observe the Unhandled Exception

## Context
Most likely this bug was introduced with Pull Request 1767
This happened when I wanted to verify the installed version and I got curious and wanted to check the change logs.

## Your Environment
OS: Microsoft Windows NT 10.0.19041.0 (Windows 10 2004 x64) in Windows Sandbox; Windows 10 21H2 x64 on the Machine itself
mRemoteNG Version: 1.77.2
.Net Runtime: windowsdesktop-runtime-6.0.1-win-x64.exe
Edition: MSI
Cmd line args:

**Comment by simonai1254** [comment]
Just noted that the other Links included in the about section are outdated as well as the develop branch does no longer exist.

**Comment by simonai1254** [comment]
Just found the faulty link opener in this file here: mRemoteNG/UI/Forms/FrmAbout.cs
It was introduced with the following pull request: 
https://github.com/mRemoteNG/mRemoteNG/pull/1767/files#diff-2c10d471afc62620b74e4290c0bfdac94a930e0284900f41c781efa9ab4a322d

**Comment by Kvarkas** [comment]
has been fixed in branch 1.77.2-dev - will be reflected in nb release
